invenio/templates/volumes.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: "v1"
+kind: "PersistentVolumeClaim"
+metadata:
+  name: "invenio-statics-claim"
+spec:
+  accessModes:
+    - "ReadWriteMany"
+  resources:
+    requests:
+      storage: "10Gi"
```

Comment:
Maybe this could prove to be a bit unelastic, too big or too small. Should this also be a Helm value?

Reply:
Done :)

Reply:
@ppanero sorry how is done? I still see the `10Gi` value hardcoded here...is github messing with me???